Monome.h

```diff
@@ -1,8 +1,22 @@
-#ifndef Monome_h
-#define Monome_h
+/*
+  Monome.h - Library for communicating with Monome devices over USB via monome serial protocol.
+  Created by Josh Ruihley, January 25, 2022.
+*/
+#ifndef MONOME_H
```

Comment:
I made this macro uppercase, following [CGL ES.32](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#Res-ALL_CAPS)